apps/cloudfront-functions/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "root": true,
+  "extends": [
+    "custom/eslint-recommended"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "**/__tests__/**/*.ts"
+      ],
+      "rules": {
+        "functional/no-return-void": "off"
+      }
+    }
+  ]
```

Comment:
I would bring the part of overrides directly to [eslint-recommended](https://github.com/pagopa/b2b-portals/blob/9e398a0/packages/eslint-config/eslint-recommended.js) since it is also duplicated in [apps/nextjs-website/.eslintrc.json](https://github.com/pagopa/b2b-portals/blob/9e398a0/apps/nextjs-website/.eslintrc.json#L7-L16). And I would remove it from the two apps.